crypto/pem/pem_lib.c

```diff
@@ -537,12 +536,13 @@ int PEM_get_EVP_CIPHER_INFO(char *header, EVP_CIPHER_INFO *cipher)
     *header = '\0';
     cipher->cipher = enc = EVP_get_cipherbyname(dekinfostart);
     *header = c;
+    header++;
```

Comment:
@richsalz The original code looked wrong because it looked like the new value of `header` wasn't used anywhere. I could have come to exactly the same wrong conclusion if I saw `*header++` there.